t/mojo/reactor_poll.t

```diff
@@ -7,332 +7,374 @@ use IO::Socket::INET;
 use Mojo::Reactor::Poll;
 use Mojo::Util qw(steady_time);
 
-# Instantiation
-my $reactor = Mojo::Reactor::Poll->new;
-is ref $reactor, 'Mojo::Reactor::Poll', 'right object';
-is ref Mojo::Reactor::Poll->new, 'Mojo::Reactor::Poll', 'right object';
-undef $reactor;
-is ref Mojo::Reactor::Poll->new, 'Mojo::Reactor::Poll', 'right object';
-use_ok 'Mojo::IOLoop';
-$reactor = Mojo::IOLoop->singleton->reactor;
-is ref $reactor, 'Mojo::Reactor::Poll', 'right object';
-
-# Make sure it stops automatically when not watching for events
-my $triggered;
-Mojo::IOLoop->next_tick(sub { $triggered++ });
-Mojo::IOLoop->start;
-ok $triggered, 'reactor waited for one event';
-my $time = time;
-Mojo::IOLoop->start;
-Mojo::IOLoop->one_tick;
-ok time < ($time + 10), 'stopped automatically';
-
-# Listen
+subtest "Instantiation" => sub {
+  my $reactor = Mojo::Reactor::Poll->new;
+  is ref $reactor, 'Mojo::Reactor::Poll', 'right object';
+  is ref Mojo::Reactor::Poll->new, 'Mojo::Reactor::Poll', 'right object';
+  undef $reactor;
+  is ref Mojo::Reactor::Poll->new, 'Mojo::Reactor::Poll', 'right object';
+  use_ok 'Mojo::IOLoop';
+  $reactor = Mojo::IOLoop->singleton->reactor;
+  is ref $reactor, 'Mojo::Reactor::Poll', 'right object';
+
+};
+
+my $reactor = Mojo::IOLoop->singleton->reactor;
```

Comment:
This seems like completely new code that changes what the test does.

Reply:
These tests had far too much shared code, which is why so much of it had to be moved out of the subtests.  I was hoping to discuss how to better compartmentalize the tests.  

But then I got into a fugue state of converting more and more -- only to find I was breaking the tests when run on integration, as I was missing local ENV vars to run all the tests.  so it goes.